VideoWeb/VideoWeb.Common/Models/ParticipantStatus.cs

```diff
@@ -14,4 +14,14 @@ public enum ParticipantStatus
         InConsultation = 6,
         Disconnected = 7
     }
+    
+    /// <summary>
+    /// Known states of an endpoint
+    /// </summary>
+    public enum EndpointStatus
+    {
```

Comment:
Why does the participant status model have endpoint status Shaed?

Reply:
It's just in the same file. I'll move it out into it's own file